daml/Exberry/Integration.daml

```diff
@@ -48,6 +49,14 @@ template NewOrderFailure
   where
     signatory integrationParty
 
+template NewOrderCancelled
```

Comment:
I think the intent behind the name was `New (Order-Cancelled)` rather than `(New-Order) Cancelled`. Agreed it can be easy to misread, though (I saw it the latter way at first as well)

Reply:
Actually, it's not immediately clear to me how `NewOrderCancelled` differs from `OrderCancelSuccess`... seems like they both come from exberry to denote a cancelled order. Would both contracts get created for one cancelled order?

Reply:
`OrderCancelSuccess` is a response from the order cancel endpoint request, whereas this comes from the streaming API and could be any order from the book, as far as I can tell.

Reply:
Not 100% sure if both would get created from a cancel request, but I think this one could still be necessary in case of an order getting canceled from another system or the Exberry UI.

Reply:
This request was added as the Marketplace needs to know when Market orders haven't executed, specifically the "Fill or Kill" market order request which is either fully executed or not at all -  

> Order Cancel Message indicates that an order on the book is being cancelled. This message also sent in case of market order that was not fully filled.

As for the naming of the template, I just followed the naming standard of the other templates - but I agree, it could be better!